src/RA_Integration.vcxproj

```diff
@@ -195,11 +194,15 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <TargetName>RA_Integration</TargetName>
+    <CodeAnalysisRuleSet>NativeRecommendedRules.ruleset</CodeAnalysisRuleSet>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
```

Comment:
I had it on and switched it back off so I could change the toolset back to v141_xp. I was just checking if the our annotated functions were matching the specification.